opentelemetry/proto/profiles/v1/alternatives/arrays/arrays.proto

```diff
@@ -0,0 +1,169 @@
+syntax = "proto3";
+
+package opentelemetry.proto.profiles.v1.alternatives.arrays;
+
+import "opentelemetry/proto/common/v1/common.proto";
+import "opentelemetry/proto/resource/v1/resource.proto";
+
+option go_package = "go.opentelemetry.io/proto/otlp/profiles/v1/arrays";
+
+
+// A pointer from a profile to a trace span.
+message Link {
+  // A unique identifier of a trace that this linked span is part of. The ID is a
+  // 16-byte array.
+  bytes trace_id = 1;
+  // A unique identifier for the linked span. The ID is an 8-byte array.
```

Comment:
```suggestion
  // A unique identifier for the linked span. The ID is an 16-byte array.
```

Similar to the `trace_id` a 16 byte array should be preferred over a 8-byte array for global uniqueness.